### ISSUE TITLE
Too hard to install fabric8 ...It's too difficult to install fabric8

### ISSUE THREAD

**Issue by damozhiying** [fabric8io/fabric8#7101]
please see issues fabric8 author ,there too many issues about install fabri8
![image](https://user-images.githubusercontent.com/10277950/45791510-87062300-bcbc-11e8-8db7-6442a7abc86d.png)

**Comment by damozhiying** [comment]
![image](https://user-images.githubusercontent.com/10277950/45810708-d15bc400-bcfd-11e8-9a03-84800ca3077f.png)

i had deploy fabric8 in minikube ,openshift,   there no one is successed,        the fabric8 is too ugly

**Comment by damozhiying** [comment]
sudo ./gofabric8 deploy --package system --http=true --legacy=false -n fabric8  --ingress=false --github-client-id=xxx --github-client-secret=xx  --legacy=true --api-server=node11 -y

**Comment by damozhiying** [comment]
@chmouel  the fabric8 project is deprecated,there are too many issues

**Comment by Mayershi** [comment]
@damozhiying   Me, too !

**Comment by chmouel** [comment]
I agree and I have unfortunately not an answer for you, in the meantime you
may want to look at jenkins-x http://jenkinsx.io
